cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -266,28 +267,24 @@ def upload_rna_coverage_bigwig_to_scout(self, case_id: str, dry_run: bool) -> No
 
             LOG.info(f"RNA coverage bigwig file {rna_coverage_bigwig.path} found.")
             for dna_case_id in rna_dna_collection.dna_case_ids:
-                if self.status_db.get_case_by_internal_id(dna_case_id).is_uploaded:
-                    LOG.info(
-                        f"Uploading RNA coverage bigwig file for sample {dna_sample_name} "
-                        f"in case {dna_case_id} in Scout."
-                    )
+                LOG.info(
+                    f"Uploading RNA coverage bigwig file for sample {dna_sample_name} "
+                    f"in case {dna_case_id} in Scout."
+                )
```

Comment:
Is this needed? I think the log info below conveys the same information. (i..e the uploaded)
In the test screenshot it also states multiple times that something is finished, maybe we can combine them?

Reply:
Yeah, I think potentially that the "Uploading" is conveying more information than the "uploaded" statement, though. If all uploads succeed, we get an "uploaded" summary with this change, but if it crashes, the "uploading" statement will tell us for which sample (although maybe the stacktrace helps as well).